UIDokan/Sources/UIDokan/Extentions/Colors+Style.swift

```diff
@@ -1,3 +1,12 @@
+import UIKit.UIColor
+
+typealias Color = UIColor
+
+// MARK: - Colors
+
+extension Color {
+      static let white = Color(named: "White")!
```

Comment:
static computed property would be much better to avoid storing classes in memory.

Reply:
Done